### ISSUE TITLE
Change IPAddressField to GenericIPAddressField for Django 1.9 Support

### ISSUE THREAD

**Issue by wyc** [istrategylabs/django-wordpress#18]
Warning during `django-admin migrate`:

```
System check identified some issues:

WARNINGS:
wordpress.Comment.author_ip: (fields.W900) IPAddressField has been deprecated. Support for it (except in historical migrations) will be removed in Django 1.9.
    HINT: Use GenericIPAddressField instead.
```

**Comment by kopf** [comment]
Is this project dead?

**Comment by collinanderson** [comment]
@kopf see #21

**Comment by kopf** [comment]
Aha! Thanks.

**Comment by jcarbaugh** [comment]
Fixed in #20
